03week/airplane/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  <title>UX Atlantic | Home Page</title>
+</head>
+<body>
+  <header>
+    <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/logo.png">
+    <nav>
+      <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/home-icon.png">
+      <a href="/home" class="home">Home</a>
+      <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/deal-icon.png">
+      <a href="/deals">Travel Deals</a>
+      <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/schedule-icon.png">
+      <a href="/schedule">Flight Schedule</a>
+      <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/login-icon.png">
+      <a href="/register">Register/Login</a>
+    </nav>
+  </header>
+  <div class="booking-menu">
+    <div class="booking-header">
+      <ul>
+        <li class="plane1">
+          <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/plane-icon.png" class="plane">
+        </li>
+        <li class="hotel1">
+          <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/hotel-icon.png" class="hotel">
+        </li>
+        <li class="car1">
+          <img src="https://raw.githubusercontent.com/AustinCodingAcademy/ciw-workbook/master/Week-3/Team-Project-3/car-icon.png" class="car">
+        </li>
+      </ul>
+    </div>
+
+    <form action="/action">
+      <input type="text" name="Depart" value="Depart">
+      <input type="text" name="Arrive" value="Arrive"><br>
+      <input type="text" name="date" value="mm/dd/yyyy">
+      <input type="text" name="date" value="mm/dd/yyyy">
+      <p class="passengers">Passengers</p>
+      <input type="text" name="date" value="1" id="number">
+
+      <!--<select name="passengers">
```

Comment:
curious why you used text input instead of select.

Reply:
I couldn't figure out how to style a select element, so I gave up and styled a text element so I could have it looking roughly like the example at least.